dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/cache/impl/TaskInstanceCacheManagerImplTest.java

```diff
@@ -0,0 +1,160 @@
+package org.apache.dolphinscheduler.server.master.cache.impl;
```

Comment:
You need to add Apache license header here, you can find the license header in other file.
And when you add an ut file, you need to add the test class in root pom.xml like below:
```java
<include>**/api/service/MonitorServiceTest.java</include>
<include>**/api/service/ProcessDefinitionServiceTest.java</include>
<include>**/api/service/ProcessTaskRelationServiceImplTest.java</include>
<include>**/api/service/TaskDefinitionServiceImplTest.java</include>
```

Reply:
ok, I've added.